src/v2/controllers/FormController.js

```diff
@@ -248,8 +248,7 @@ export default Controller.extend({
 
     if (IonResponseHelper.isIonErrorResponse(error)) {
       const convertedErrors = IonResponseHelper.convertFormErrors(error);
-      const showBanner = convertedErrors.responseJSON.errorCauses.length ? false : true;
-      model.trigger('error', model, convertedErrors, showBanner);
+      model.trigger('error', model, convertedErrors, true);
```

Comment:
nit:
looks like the `if/elseif` block can just create error object and invoke `model.trigger('error', ..` after.

Reply:
Fixed